scripts/fuzz.sh

```diff
@@ -85,10 +87,13 @@ main()
   local BUILD_ONLY=false
   local FUZZER_STDOUT=/dev/stdout # bashism
   local TEST_DURATION=3
+  local IPC
 
   # Parse "$@". getopts stops after '--'
-  while getopts "hj:po:t:b" opt; do
+  while getopts "34hj:po:t:b" opt; do
       case "$opt" in
+          3) IPC=3;;
+          4) IPC=4;;
```

Comment:
can they be made mutually-exclusive? Or make it `--ipc=4`?

Reply:
Both are possible but I think this script is growing too big. libfuzzer works well but it has been deprecated by LLVM. The more complex is this script, the more effort to maintain it.

Reply:
I changed it to `-i3` /  `-i 4`, small enough change and keeps the code simple and short. It's not actually mutually exclusive (last one still wins) but it's more obvious that you're doing something wrong.